dev-lang/mlton/mlton-20180207.ebuild

```
@@ -88,7 +88,7 @@ mlton_bootstrap_variant() {
 }
 
 mlton_bootstrap_build_dir() {
-	echo $(basename ${S})"-"$(mlton_bootstrap_variant)
+	echo $(basename "${S}")"-"$(mlton_bootstrap_variant)
```

Not your fault, but nice place: why is `-` quoted ;-)
Let's quote just for easier readability on the line

```suggestion
	echo "$(basename "${S}")-$(mlton_bootstrap_variant)"
```

---

done